cmd/influxd/main.go

```diff
@@ -78,11 +74,13 @@ func main() {
 	case "":
 		execRun(args)
 	case "backup":
+		setGoMaxProcs()
```

Comment:
Minor: why do it in the `switch` statement for some cases, but then do it explicitly in `execRun` for that case?